easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.10-intel-2022a.eb

```diff
@@ -13,31 +16,22 @@ source_urls = ['https://gitlab.com/Molcas/OpenMolcas/-/archive/v%(version)s/']
 sources = ["OpenMolcas-v%(version)s.tar.gz"]
 patches = ['OpenMolcas-21.06_mcpdft_deps.patch']
 checksums = [
-    {'OpenMolcas-v22.10.tar.gz': 'd1d3d6c9bd69486c36f5285a51ac18e75c03540c5b33751855e94abc764d9658'},
-    {'OpenMolcas-21.06_mcpdft_deps.patch': '708763e060aa0174bbe9c0cc336b509fbf492066465908dd919452836a7a793a'},
+    'd1d3d6c9bd69486c36f5285a51ac18e75c03540c5b33751855e94abc764d9658',  # OpenMolcas-v22.10.tar.gz
+    '708763e060aa0174bbe9c0cc336b509fbf492066465908dd919452836a7a793a',  # OpenMolcas-21.06_mcpdft_deps.patch
 ]
 
-builddependencies = [('CMake', '3.23.1')]
-
 dependencies = [
-    ('Python', '3.10.4'),
-    ('HDF5', '1.12.2'),
     ('GlobalArrays', '5.8.1'),
+    ('HDF5', '1.12.2'),
+    ('Python', '3.10.4'),
 ]
 
-build_shared_libs = True
+builddependencies = [('CMake', '3.24.3')]
 
 preconfigopts = 'GAROOT=$EBROOTGLOBALARRAYS'
-_copts = [
-    '-DLINALG=MKL',
-    '-DMKLROOT=$MKLROOT',
-    '-DOPENMP=ON',
-    '-DTOOLS=ON',
-    '-DFDE=ON',
-    '-DMPI=ON',
-    '-DGA=ON',
-]
-configopts = ' '.join(_copts)
+configopts = '-DLINALG=MKL -DMKLROOT=$MKLROOT '
+configopts += '-DMPI=ON -DOPENMP=ON -DGA=ON '
+configopts += '-DBUILD_SHARED_LIBS=ON -DTOOLS=ON -DFDE=ON'
```

Comment:
The option `-DBUILD_SHARED_LIBS=ON` can be removed as it is already default in CMakeMake (`build_shared_libs = True` is also default).

Reply:
I was wrong here, `build_shared_libs` is not default. I confused it with `BUILD_TYPE=Release`.